src/System.Threading.Tasks/tests/Task/ExecutionContextFlowTest.cs

```diff
@@ -30,6 +30,38 @@ public void SuppressFlow_TaskCapturesContextAccordingly(bool suppressFlow)
             }
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, reason: "Fix in .NET Core: https://github.com/dotnet/coreclr/pull/20294")]
```

Comment:
>TargetFrameworkMonikers.NetFramework, reason: "Fix in .NET Core: https://github.com/dotnet/coreclr/pull/202 [](start = 31, length = 107)

this comment looks confusing to me as we are skipping the full framework run and the comment saying the fix is on net core

Reply:
What's confusing?  The fix is in .NET Core but not in .NET Framework, so we're skipping on .NET Framework.  What would you prefer to see?

Reply:
I expect the comment is telling why we are skipping the full framework not telling we are supporting it on net core.